lnwire/channel_update.go

```diff
@@ -143,6 +147,7 @@ func (a *ChannelUpdate) Encode(w io.Writer, pver uint32) error {
 		a.MsgFlags,
 		a.TimeLockDelta,
 		a.HtlcMinimumMsat,
+		a.HtlcMaximumMsat,
```

Comment:
don't think this will work, as this field is part of what was previously opaque data.

Reply:
moved to the end. i hope that fixes this feedback, let me know if not